### ISSUE TITLE
Bring FMIS Animation Functionality into OM3

### ISSUE THREAD

**Issue by jamesdoyle21** [asascience/ott-coops#475]

**Comment by mpiannucci** [comment]
Setup circleci integration to ECR

Deploying this to AZ-NAM-MARACOOS-DEMO on port 6005, waiting on scott to open that port

**Comment by mpiannucci** [comment]
On hold for now that infra is in place... EIther Dubai or EDS will take htis one on
